implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/internal/DatadogStatsdLineBuilder.java

```diff
@@ -38,16 +38,15 @@ public class DatadogStatsdLineBuilder extends FlavorStatsdLineBuilder {
 
     private final Object conventionTagsLock = new Object();
 
-    @SuppressWarnings({ "NullableProblems", "unused" })
+    @SuppressWarnings({ "NullAway.Init", "unused" })
```

Comment:
Can we create an issue to fix these?
I think:
- `NullAway.Init` might not be needed on `namingConvention`
- `name` and `tagsNoStat` can be initialized with `""` and can be `@NonNull`

Same for the other *LineBuilders.

Reply:
Perhaps. I didn't want to mess with the logic to get the migration done when we have no reported issues for these. It's probably best to try to improve that in a separate change.

Reply:
I created a PR: https://github.com/micrometer-metrics/micrometer/pull/6285